src/main/telemetry/smartport.c

```diff
@@ -42,6 +42,7 @@
 #include "sensors/battery.h"
 #include "sensors/acceleration.h"
 #include "sensors/barometer.h"
+#include "sensors/pitotmeter.h"
```

Comment:
Lots of unnecessary `#include`s of `pitometer.h`. These should only be added when the code that uses them is added. Only "include what you use" is the principle here. As well as being good practice, there is the practical issue that unnecessary `#includes` slow down the build, and `make all` takes long enough already.